src/components/Calculator.js

```diff
@@ -12,32 +12,35 @@ const Calculator = () => {
-        <button type="button" className="grid-button number" onClick={clickHandle}>1</button>
-        <button type="button" className="grid-button number" onClick={clickHandle}>2</button>
-        <button type="button" className="grid-button number" onClick={clickHandle}>3</button>
-        <button type="button" className="grid-button operator column-4" onClick={clickHandle}>+</button>
-        <button type="button" className="grid-button number" id="zero" onClick={clickHandle}>0</button>
-        <button type="button" className="grid-button number" onClick={clickHandle}>.</button>
-        <button type="button" className="grid-button operator column-4" onClick={clickHandle}>=</button>
+    <div className="container">
+      <h2>Let&apos;s do some math!</h2>
+      <div className="calculator-container">
+        <div className="display" id="display">
+          {total}
+          {operation}
+          {next}
+        </div>
+        <div className="grid-container" id="button-grid">
+          <button type="button" className="grid-button operator" onClick={clickHandle}>AC</button>
+          <button type="button" className="grid-button operator" onClick={clickHandle}>+/-</button>
+          <button type="button" className="grid-button operator" onClick={clickHandle}>%</button>
+          <button type="button" className="grid-button operator column-4" onClick={clickHandle}>÷</button>
+          <button type="button" className="grid-button number" onClick={clickHandle}>7</button>
+          <button type="button" className="grid-button number" onClick={clickHandle}>8</button>
+          <button type="button" className="grid-button number" onClick={clickHandle}>9</button>
+          <button type="button" className="grid-button operator column-4" onClick={clickHandle}>x</button>
+          <button type="button" className="grid-button number" onClick={clickHandle}>4</button>
+          <button type="button" className="grid-button number" onClick={clickHandle}>5</button>
+          <button type="button" className="grid-button number" onClick={clickHandle}>6</button>
+          <button type="button" className="grid-button operator column-4" onClick={clickHandle}>-</button>
+          <button type="button" className="grid-button number" onClick={clickHandle}>1</button>
+          <button type="button" className="grid-button number" onClick={clickHandle}>2</button>
+          <button type="button" className="grid-button number" onClick={clickHandle}>3</button>
+          <button type="button" className="grid-button operator column-4" onClick={clickHandle}>+</button>
+          <button type="button" className="grid-button number" id="zero" onClick={clickHandle}>0</button>
+          <button type="button" className="grid-button number" onClick={clickHandle}>.</button>
+          <button type="button" className="grid-button operator column-4" onClick={clickHandle}>=</button>
```

Comment:
- [ ] Great work you have done so far on this project. However, this milestone was about testing your application using Jest and React testing Library and I have not seen any test code in a `_.test.js` file. Please see [project requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md). 
* **The test should cover:** 
  - All the functionalities & scenarios of the `operate.js` and `calculate.js` modules should be covered using Jest.
  - There should be the usage of Jest snapshots for testing all the components.
  - There should be the usage of React Testing Library to simulate user interaction.

Please implement this a re-submit for review.

Reply:
@bobb-Rob 
This submission was done by mistake. It's not for the test. I'll create another PR for that. Thank you.